src/components/Grants/GrantCard.tsx

```diff
@@ -34,20 +32,27 @@ export const PROPOSAL_GRANT_CATEGORY_COLORS: Record<ProposalGrantCategory, PillC
 
 const GrantCard = ({ grant }: GrantCardProps) => {
   const category: ProposalGrantCategory = grant.configuration.category
+  const token = grant.contract ? grant.contract.symbol : grant.enacting_token
```

Comment:
Here we would just use the `grant.token` we got from transparency right? 
( the `TransparencyGrant` ;) )